globalize-compiler/globalize-compiler.d.ts

```diff
@@ -0,0 +1,106 @@
+// Type definitions for globalize-compiler v0.2.0
+// Project: https://github.com/jquery-support/globalize-compiler
+// Definitions by: Ian Clanton-Thuon <https://github.com/iclanton>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="./../globalize/globalize.d.ts" />
+/// <reference path="../estree/estree.d.ts" />
+
+declare namespace GlobalizeCompiler {
```

Comment:
s/GlobalizeCompiler/globalizeCompiler/
You should use cameCase about namespace.